docs/contributing.md

```diff
@@ -73,6 +77,62 @@ To run it locally:
 pre-commit run --all-files
 ```
 
+
+## Building TA with the Local Version of UCC
+
+UCC is a tool for Technical Add-ons (TAs), so it's important to test TA generation while developing UCC locally.
+
+### Overview
```

Comment:
This would look very similar to the changes made in #1393, should we wait for it to be merged and cross-link it?

Reply:
It is not. I added an instruction how to test TA with local version of UCC (aka test UCC for a certain add-on).
In #1393 there is an instruction how to install UCC from pypi